packages/strapi-plugin-documentation/services/Documentation.js

```diff
@@ -402,6 +412,7 @@ module.exports = {
       }
 
       const verbObject = {
+        operationId: controllerMethod + controllerName,
```

Comment:
@alexandrebodin I'm not sure, is it unique ? As example it gives `updateCountry`

Reply:
There are other parts of the code were operationId should be added. I thing line 685 for instance.
I launched and they all have an operationId except the plugins

Reply:
I fyou have two controllers in two different apis with the same method and names this will not be unique indeed

Reply:
But it would break all the app anyway today no ?